libcore/PluginManager.vala

```diff
@@ -224,9 +224,17 @@ public class Marlin.PluginManager : Object {
             plugin.update_sidebar (widget);
     }
 
-    public void update_file_info (GOF.File file) {
-        foreach (var plugin in plugin_hash.values)
+    public void update_file_info (GOF.File? file) {
```

Comment:
Shouldn't that method have the file not nullable by design? Every object that calls this should just verify if the file is actually a non-null object. Why is that change?

Reply:
It seemed more efficient to check once here rather than having to all calling objects ensure that no null files are passed.